.pre-commit-config.yaml

```diff
@@ -10,9 +10,8 @@ repos:
   - repo: local
     hooks:
       - id: check-dir-names
-        name: Check snake_case directory names
+        name: Check snake_case directory and file names
         entry: ./.hooks/check_dir_names.py
         language: python
-        exclude: ^src/addons
 
-exclude: ^src/addons
+exclude: '^src/addons/|^README\.md$|^LICENSE$|^.pre-commit-config.yaml$'
```

Comment:
Not a problem for right now but this exclude applies to all the hooks not just your new one.

This will matter in my own branch so will fix it then